src/ecs/commands/runCommandInContainer.ts

```diff
@@ -87,3 +116,35 @@ export async function runCommandInContainer(
         status?.dispose()
     }
 }
+/**
+ * Attempts to verify if the task role provided has the required SSM permissions to run ECS Exec
+ * @param taskRoleArn
+ * @param iamClient
+ * @returns True if the permissions are missing from the Task role. False when all required permissions were found. Undefined when the task role is missing or the 'simulatePrincipalPolicy' call was unsuccessful.
+ */
+export async function isMissingRequiredPermissions(
```

Comment:
"is missing" is negative logic. Usually it makes sense to have positive logic. 
```suggestion
export async function hasEcsExecPermissions(
```